doc/source/developing/testing.rst

```diff
@@ -3,503 +3,400 @@
-downloaded from https://yt-project.org/data/. We do not explicitly list the
-datasets we use in the tests here because the list of necessary datasets changes
-regularly, instead you should take a look at the tests you would like to run and
-make sure that the necessary data files are downloaded before running the tests.
+To run the answer tests locally, you must first generate a set of gold standard answer files using a "known good" version of yt. Once done, then update to the version of yt you want to test and
+run the tests again so that the results can be compared to those contained in the gold standard answer files.
 
-To run the answer tests, you must first generate a set of test answers locally
-on a "known good" revision, then update to the revision you want to test, and
-run the tests again using the locally stored answers.
+As an example, let's focus on running the answer tests for the tipsy frontend.
 
-Let's focus on running the answer tests for a single frontend. It's possible to
-run the answer tests for **all** the frontends, but due to the large number of
-test datasets we currently use this is not normally done except on the yt
-project's contiguous integration server.
+.. note::
+    It's possible to run the answer tests for **all** the frontends, but due to the large number of test datasets we currently use this is not normally done except on the yt project's contiguous integration server.
 
 .. code-block:: bash
 
    $ cd $YT_GIT
-   $ nosetests --with-answer-testing --local --local-dir $HOME/Documents/test --answer-store --answer-name=local-tipsy yt.frontends.tipsy
+   $ pytest --with-answer-testing --answer-store --local-dir="$HOME/Documents/test" -k "TestTipsy"
 
-This command will create a set of local answers from the tipsy frontend tests
-and store them in ``$HOME/Documents/test`` (this can but does not have to be the
-same directory as the ``test_data_dir`` configuration variable defined in your
-``~/.config/yt/yt.toml`` file) in a file named ``local-tipsy``. To run the tipsy
-frontend's answer tests using a different yt changeset, update to that
-changeset, recompile if necessary, and run the tests using the following
-command:
+This command will run the tipsy answer tests and, because of the presence of the ``--answer-store`` option, save the results in a local gold standard answer file in ``$HOME/Documents/test``.
 
-.. code-block:: bash
+.. note::
+    The path specified by ``--local-dir`` can, but does not have to be, the same directory as the ``test_data_dir`` configuration variable.
+
+The gold standard answer file will be named ``tipsy_answers_xyz.yaml``, where ``xyz`` denotes the version number of the gold standard answers.
 
-   $ nosetests --with-answer-testing --local --local-dir $HOME/Documents/test --answer-name=local-tipsy yt.frontends.tipsy
+.. note::
+    The answer version number is determined by the ``answer_version`` attribute of the class being tested (e.g., ``TestTipsy.answer_version``).
```

Comment:
This doesn't need to be a note, I think. Add it as a sentence after the previous line.